code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -5,791 +5,791 @@
+	result = /obj/item/food/snacks/donut/jelly/slimejelly
 
 /datum/recipe/microwave/jellydonut/cherry
 	reagents = list("cherryjelly" = 5, "sugar" = 5)
 	items = list(
-		/obj/item/reagent_containers/food/snacks/dough
+		/obj/item/food/snacks/dough
 	)
-	result = /obj/item/reagent_containers/food/snacks/donut/jelly/cherryjelly
+	result = /obj/item/food/snacks/donut/jelly/cherryjelly
 
 /datum/recipe/microwave/donut
 	reagents = list("sugar" = 5)
 	items = list(
-		/obj/item/reagent_containers/food/snacks/dough
+		/obj/item/food/snacks/dough
 	)
-	result = /obj/item/reagent_containers/food/snacks/donut
+	result = /obj/item/food/snacks/donut
 
 /datum/recipe/microwave/donut/sprinkles
 	reagents = list("sugar" = 5, "sprinkles" = 2)
 	items = list(
-		/obj/item/reagent_containers/food/snacks/dough
+		/obj/item/food/snacks/dough
 	)
-	result = /obj/item/reagent_containers/food/snacks/donut/sprinkles
+	result = /obj/item/food/snacks/donut/sprinkles
 
 /datum/recipe/microwave/human/burger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/meat/human,
-		/obj/item/reagent_containers/food/snacks/bun
+		/obj/item/food/snacks/meat/human,
+		/obj/item/food/snacks/bun
 	)
-	result = /obj/item/reagent_containers/food/snacks/human/burger
+	result = /obj/item/food/snacks/human/burger
 
 /datum/recipe/microwave/plainburger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/bun,
-		/obj/item/reagent_containers/food/snacks/meat //do not place this recipe before /datum/recipe/microwave/humanburger
+		/obj/item/food/snacks/bun,
+		/obj/item/food/snacks/meat //do not place this recipe before /datum/recipe/microwave/humanburger
 	)
-	result = /obj/item/reagent_containers/food/snacks/burger/plain
+	result = /obj/item/food/snacks/burger/plain
 
 /datum/recipe/microwave/syntiburger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/bun,
-		/obj/item/reagent_containers/food/snacks/meat/syntiflesh
+		/obj/item/food/snacks/bun,
+		/obj/item/food/snacks/meat/syntiflesh
 	)
-	result = /obj/item/reagent_containers/food/snacks/burger/plain
+	result = /obj/item/food/snacks/burger/plain
 
 /datum/recipe/microwave/brainburger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/food/snacks/bun,
 		/obj/item/organ/internal/brain
 	)
-	result = /obj/item/reagent_containers/food/snacks/burger/brain
+	result = /obj/item/food/snacks/burger/brain
 
 /datum/recipe/microwave/hamborger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/food/snacks/bun,
 		/obj/item/robot_parts/head
 	)
-	result = /obj/item/reagent_containers/food/snacks/burger/hamborger
+	result = /obj/item/food/snacks/burger/hamborger
 
 /datum/recipe/microwave/xenoburger
 	items = list(
-		/obj/item/reagent_containers/food/snacks/bun,
-		/obj/item/reagent_containers/food/snacks/monstermeat/xenomeat	)
-	result = /obj/item/reagent_containers/food/snacks/burger/xeno
+		/obj/item/food/snacks/bun,
+		/obj/item/food/snacks/monstermeat/xenomeat	)
```

Comment:
could you kill the whitespace before right paren here?

Reply:
All cleanup reviews adressed, thanks!